CHANGELOG.md

```diff
@@ -2,193 +2,182 @@
 
 All notable changes to this project will be documented in this file.
 
-# Unreleased
+# Capablanca Release
 
-## Sprint 28 end 2015-08-24
-
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+28+Ending+Aug+24%2C+2015%22+label%3Aenhancement)
-
-- REST API
- - Roles CRUD
- - Product features collection
- - Chargeback Rates CRUD
- - Reports run action
- - Report results collection
-- Authentication
- - External Auth to AD web ui login & SSO - Manual configuration
- - Extenral Auth to LDAP - Manual configuration
-- UI
- - Orchestration: Stacks Retirement added
- - Tenants added to Access Control (pr should be merged today)
- - Self Service UI - initial PR (some teaser screen shots)
-- SmartState Analysis
- - Refactored the middle layer of the SmartState Analysis stack
-- Providers
- - Namespacing Providers: OpenStack, Containers
- - Azure Provider models
- - Containers: Kubernetes + RHEV
- - OpenStack: shelve VMs
-
-### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+28+Ending+Aug+24%2C+2015%22+label%3Aenhancement)
-
-- Providers: Moved provider event filters into the database (blacklisted events)
-- Provisioning: Filter Service Catalog Items during deployment
-- UI
- - Retirement screens converted to Angular
- - DHTMLX menus replaced with Bootstrap/Patternfly menus
- - Host editor converted to Angular
-- Appliance
- - PostgreSQL 9.2 => 9.4.1
- - Cloud-init added back.  Reported issues have been addressed
- - Changed file system from ext4 to xfs
-
-- Gem updates
- - Upgraded rufus scheduler to version 3
- - Upgraded to latest net-sftp
- - Upgraded to latest net-ssh  
- - Upgraded to latest ruby-progressbar
- - Upgraded to latest snmp
-
-### [Fixed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+28+Ending+Aug+24%2C+2015%22+label%3Abug)
-
-Notable fixes include:
-- VMware snapshot from automate fixed - memory parameter added
-
-## Sprint 27 end 2015-08-03
+## Added Features
 
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+27+Ending+August+3%2C+2015%22+label%3Aenhancement)
+### Providers
+- Containers
+ - Kubernetes and RHEV Integration
+ - Container Integration: Kubernetes Events
+ - Topology Widget
+ - Kubernetes and VMware integration
+-  Namespacing
+  - Preparation for pluggable providers
+  - OpenStack, Containers
```

Comment:
I think we did a lot more than just OpenStack and Containers...I think we did *everything*